src/App.tsx

```diff
@@ -46,6 +53,9 @@ function App() {
     const value: string = e.target.value;
     if (value === "AC" || value === "C") {
       // Handles Clear and All Clear
+      // Removes active operator style
+      if (operation.current)
+        operatorRefs.get(operation.current)!.current.style.border = "";
```

Comment:
TODO: This code does not need to immediately follow `if (value === "AC" || value === "C") {`. Place inside appropriate block as part of the [refactoring and modularization issue](https://github.com/vsots/calculator/issues/3)

Reply:
[Done](https://github.com/vsots/calculator/pull/4/files#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2L58)